client/state/themes/actions.js

```diff
@@ -418,6 +421,51 @@ export function clearActivated( siteId ) {
 }
 
 /**
+ * Triggers a network request to activate a specific wpcom theme on a given Jetpack site.
+ * First step of the process is the installation of the theme on Jetpack site.
+ * Second step is the actuall activation.
+ *
+ * @param  {Number|String} siteId    Site ID
```

Comment:
We've had problems using strings in the past because of the :: in some jetpack site slugs. Best to stick to `Number`.